projects/npm-tools/packages/npm-scripts/src/utils/getDXPVersion.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const findRoot = require('./findRoot');
+
+/**
+ * Attempts to infer the current DXP version based on the contents of the
+ * `release.properties` file in the repo root directory.
+ */
+function getDXPVersion() {
```

Comment:
I hadn't thought of doing something like this and checking against dxp itself. Great idea!